.env.example

```diff
@@ -1,2 +1,2 @@
-REACT_APP_API_TOKEN=token
+REACT_APP_API_TOKEN="7f5db6c90c3b4afb98973cefd6d2480d"
```

Comment:
Secret values should not be stored in a Git repository because it can expose sensitive information to unauthorized users and compromise the security of the system.